modules/node-tuning-hosted-cluster.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-node-tuning-operator.adoc
+
+[id="node-tuning-hosted-cluster_{context}"]
+= Configuring node tuning in a hosted cluster (Technology Preview)
+
+//# Manage node-level tuning with the Node Tuning Operator
+
+To set node-level tuning on the nodes in your hosted cluster, you can use the Node Tuning Operator. In hosted control planes, you can configure node tuning by creating config maps that contain tuned objects and referencing those config maps in your node pools. Currently, node tuning is limited to actions that the TuneD daemon can apply directly, such as setting sysctl values. Tuning that requires setting kernel boot parameters is not yet supported for hosted control planes.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+. Create a config map that contains a valid tuned manifest, and reference the manifest in a node pool. In the following example, a tuned manifest defines a profile that sets `vm.dirty_ratio` to 55 on nodes that contain the `tuned-1-node-label` node label with any value. Save the config map manifest in a file named `tuned-1.yaml`:
+
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-1
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: tuned-1
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Custom OpenShift profile
+              include=openshift-node
+              [sysctl]
+              vm.dirty_ratio="55"
+            name: tuned-1-profile
+          recommend:
+          - priority: 20
+            profile: tuned-1-profile
+
++
+[NOTE]
+====
+If you do not add any labels to an entry in the `spec.recommend` section of the tuned spec, node-pool-based matching is assumed, so the highest priority profile in the `spec.recommend` section is applied to nodes in the pool. Although you can achieve more fine-grained node-label-based matching by setting a label value in the tuned `.spec.recommend.match` section, node labels will not persist during an upgrade unless you set the node pool `.spec.management.upgradeType` value to `InPlace`.
+====
+
+. Create the config map in the management cluster:
+   
```

Comment:
So this is a terminal command rather than yaml, which will look like this (using some of the same feedback from above). In addition, for terminal input we use a dollar sign to indicate user privileges. Sometimes we use a pound sign if different privileges are required (root users) but that's very infrequent in OpenShift:
```
. Create the config map in the management cluster: 
+
[source,terminal]
----
$ oc --kubeconfig .... .yaml
----
```